x-pack/test/fleet_api_integration/apis/agents/services.ts

```diff
@@ -5,23 +5,11 @@
  * 2.0.
  */
 
-import supertest from 'supertest';
 import { Client, HttpConnection } from '@elastic/elasticsearch';
 import { format as formatUrl } from 'url';
 
-import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
-
 import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
 
-export function getSupertestWithoutAuth({ getService }: FtrProviderContext) {
-  const config = getService('config');
-  const kibanaUrl = config.get('servers.kibana');
-  kibanaUrl.auth = null;
-  kibanaUrl.password = null;
-
-  return supertest(formatUrl(kibanaUrl));
-}
```

Comment:
Removing since it is not used in any tests, just in case you will need it in future: import service from `'@kbn/ftr-common-functional-services'`